docs.wrm/getting-started.wrm

```diff
@@ -3,14 +3,14 @@ _section: Getting Started @<getting-started>
 
 _subsection: Installing @<installing>
 
-The various Classes and Functions are available to be imported
+Ethers' various Classes and Functions are available to import
 manually from sub-packages under the [@ethersproject](link-ethers-npm)
 organization but for most projects, the umbrella package is the
 easiest way to get started.
 
 _code: @lang<shell>
 
-/home/ricmoo> npm install --save ethers@next
```

Comment:
@ricmoo This didn't work, is this change correct?

Reply:
Oh! The `@next` needs to be removed now that v5 is no longer beta.